Gulpfile.js

```diff
@@ -318,54 +301,21 @@ gulp.task('clean', function (cb) {
 });
 
 
-gulp.task("watch", ['all'], function () {
-    gulp.watch('src/**/*.hbs', ['templates']);
+gulp.task('watch', ['package'], function () {
+    gulp.watch(['src/**/*.hbs', 'src/**/*.js'], ['bundle']);
     gulp.watch('style/**/*.*', ['styles', 'package']);
     gulp.watch('*.html', ['package']);
-    gulp.watch('src/**/*.js', ['compile']);
-    gulp.run('serve')
+    gulp.run('webserver');
 });
 
-
 gulp.task('bundle', ['compile', 'templates'], function () {
-    var dist = [];
-    _.each(SOURCES, function (it) {
-        dist.push(it.replace('src/', 'dist/'))
-    })
-
-    return gulp.src(["dist/**/*.js", "dist/templates/*.js"])
-        .pipe(resolve())
+    return gulp.src(['dist/' + pkg.name + '.js', 'dist/templates/*.js'])
         .pipe(sourcemaps.init({loadMaps: true}))
-        .pipe(gulpif(prod, uglify({mangle: false})))
```

Comment:
how has the "--env production" option been handled to only uglify in prod handled?